src/compiler/builder.ts

```diff
@@ -2395,7 +2395,10 @@ export function getBuildInfoFileVersionMap(
     program: IncrementalBuildInfo,
     buildInfoPath: string,
     host: Pick<ReadBuildProgramHost, "useCaseSensitiveFileNames" | "getCurrentDirectory">,
-) {
+): {
+    fileInfos: Map<Path, string>;
+    roots: Map<Path, Path | undefined>;

```

Comment:
For cases like these where we've returned an object, I do wonder if we should be naming the type.

Reply:
It certainly would make things look better in some locations, but there are lots of places where the return type is just used as that return